orient_express/vertex.py

```
@@ -287,27 +288,33 @@ def get_vertex_model(
-        if int(model.version_id) == version:
-            return VertexModel(model, model_name, project_name, region, version)
-
-    if raise_exception:
-        raise Exception(
-            f"Model '{model_name}' with version '{version}' not found in registry for project '{project_name}' region '{region}'"
-        )
-    return None
+    if version is not None:
+        try:
+            model = aiplatform.Model(model_name=resource_name, version=str(version))
+        except Exception:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)